src/InfoViz/Native/ParallelCoordinates/index.js

```diff
@@ -841,6 +850,24 @@ function parallelCoordinate(publicAPI, model) {
     }
   });
 
+  if (model.provider.isA('DataUpdateProvider')) {
+    model.updateSubscription = model.provider.subscribeToDataUpdate(
+      'histogram2d',
+      data => {
+        Object.keys(data).forEach(xName => {
+          Object.keys(data[xName]).forEach(yName => {
+            const histoPayload = data[xName][yName][0];
+            if (histoPayload.role && histoPayload.role.type && histoPayload.role.type === 'alldata') {
+              model.provider.setHistogram2D(xName, yName, histoPayload);
```

Comment:
Because David's new push notifications include static histograms, I think it could make more sense to just get all histograms (for static as well as dynamic fields) from those notifs.  Would it be less odd if I just stopped using the Histogram2DProvider in PC altogether?